lib/entities/console.rb

```diff
@@ -0,0 +1,100 @@
+class Console
+  include Database
+  # include Statistics
+  include Rules
+  include Respondent
+
+  AVAILABLE_ACTIONS = {
+    start: 'start',
+    rules: 'rules',
+    stats: 'stats',
+    leave: 'exit',
+    save_player: 'y'
+  }.freeze
+  AVAILABLE_DIFFICULTY = %w[
+    easy
+    hard
+    expert
+  ].freeze
+
+  def initialize(process_helper: ProcessHelper.new, set_game: Game.new, statistics: Statistics.new)
+    @process_helpers = process_helper
+    @game = set_game
+    @stats = statistics
+  end
+
+  def choose_action
+    greeting
+    loop do
+      for_choose_action
+      case input
+      when AVAILABLE_ACTIONS[:start] then return process
+      when AVAILABLE_ACTIONS[:rules] then show_rules
+      when AVAILABLE_ACTIONS[:stats] then statistics
+      else wrong_input_action
+      end
+    end
+  end
+
+  private
+
+  def process
+    @player = @process_helpers.setup_player
+    @difficulty = @process_helpers.setup_difficulty
+    set_game_options(@difficulty, @player)
+    play_game
+  end
+
+  def set_game_options(difficulty, player_object)
+    @game.game_options(user_difficulty: difficulty, player: player_object)
+  end
+
+  def play_game
+    in_process
+    loop do
```

Comment:
maybe not `loop`?
maybe `while validate_game_state`

or if we rename `validate_game_state` to `game_state_valid?` or smth like that